app/views/service_manual/service_manual_topic.html.erb

```diff
@@ -1,4 +1,7 @@
 <%= content_for :title, "#{@content_item.title} - Service Manual" %>
```

Comment:
I think you need ` - GOV.UK` on the end of the title

Reply:
Thanks, have added.